dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileDataWriter.kt

```diff
@@ -76,9 +80,28 @@ internal open class BatchFileDataWriter<T : Any>(
 
     @WorkerThread
     private fun writeData(byteArray: ByteArray): Boolean {
-        val file = fileOrchestrator.getWritableFile(byteArray.size) ?: return false
+        if (!checkEventSize(byteArray.size)) return false
```

Comment:
Means that this event will be lost ? I wonder if this will create problems for SessionReplay records 🤔 , I guess they are not that big though.

Reply:
Yes, it will be lost. But this behavior is not different from the current one. Currently if item is too big we won't provide any file for writing it https://github.com/DataDog/dd-sdk-android/blob/e5ddff44deff481cebafcb6b735938fb9fe04998/dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestrator.kt#L48-L57